src/schema/sources.ts

```diff
@@ -347,56 +374,117 @@ const sourceToGQL = (source: Source): GQLSource => ({
 });
 
 export enum SourcePermissions {
-  View,
-  Post,
-  Leave,
-  Delete,
-  Edit,
+  View = 'view',
+  Post = 'post',
+  PostLimit = 'post_limit',
+  PostDelete = 'post_delete',
+  MemberRemove = 'member_remove',
+  ModeratorAdd = 'moderator_add',
+  ModeratorRemove = 'moderator_remove',
+  InviteDisable = 'invite_disable',
+  Leave = 'leave',
+  Delete = 'delete',
+  Edit = 'edit',
 }
 
+const memberPermissions = [
+  SourcePermissions.View,
+  SourcePermissions.Post,
+  SourcePermissions.Leave,
+];
+const moderatorPermissions = [
+  ...memberPermissions,
+  SourcePermissions.PostDelete,
+  SourcePermissions.MemberRemove,
+  SourcePermissions.Edit,
+];
+const ownerPermissions = [
+  ...moderatorPermissions,
+  SourcePermissions.PostLimit,
+  SourcePermissions.ModeratorAdd,
+  SourcePermissions.ModeratorRemove,
+  SourcePermissions.InviteDisable,
+  SourcePermissions.Delete,
+].filter((permission) => permission !== SourcePermissions.Leave);
```

Comment:
Why is this filter needed, did we not say owners can simply leave for now? (not sure though)

Reply:
Yes, they either delete the Squad or if in the future, pass ownership.

Reply:
But why do we filter the permission then, we simply allow deletion now right?

Reply:
Yes that's supposed to happen as far as I know:
https://dailydotdev.slack.com/archives/C03QZBF46JC/p1676535540610609?thread_ts=1676530981.133169&cid=C03QZBF46JC